lib/stats/series.js

```diff
@@ -0,0 +1,209 @@
+import base from 'taskcluster-base';
+
+const BASE_WORKER_SCHEMA = {
+    workerId: base.stats.types.String,
+    workerType: base.stats.types.String,
+    workerGroup: base.stats.types.String,
+    provisionerId: base.stats.types.String
+};
+
+export const workerStart = new base.stats.Series({
+  name: 'worker.start',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time worker started
+    value: base.stats.types.Number
+  })
+});
+
+export const workerShutdown = new base.stats.Series({
+  name: 'worker.shutdown',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
+
+export const workerUptime = new base.stats.Series({
+  name: 'worker.uptime',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
+
+
+export const workerTotalMemory = new base.stats.Series({
+  name: 'worker.memory_total',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
+
+export const workerFreeMemory = new base.stats.Series({
+  name: 'worker.memory_free',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
+
+export const workerHDUsed = new base.stats.Series({
+  name: 'worker.hd_used',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
+
+export const workerHDFree = new base.stats.Series({
+  name: 'worker.hd_free',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
+
+export const workerHDTotal = new base.stats.Series({
+  name: 'worker.hd_total',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time when worker received shutdown signal
+    value: base.stats.types.Number
+  })
+});
```

Comment:
No we efficiently query cross series... If say we wanted to look at < 50% free disk space...
I'm not sure how fast that would be in the first place, but it certainly would be very very slow, if they are in different series.

Note, I'm using the multiple values in single series for other components. 

----
Actually, I'm note sure... 

But [let's avoid `.` in series names so we don't have to quote them in queries](http://influxdb.com/docs/v0.9/concepts/schema_and_data_layout.html)...